File: Spring Boot version/src/main/java/Thalassa/Models/ConstantsData.java
package Thalassa.Models;

import com.arangodb.springframework.annotation.Document;

// This class handles essential data (stored in database) needed during service initialization.
@Document("Constants")
public class ConstantsData {

    public String certificate;
    public String certificateFileName;
    public String certificatePassword;
    public String jwtKey;

    public String getCertificate() {
        return certificate;
    }

    public void setCertificate(String certificate) {
        this.certificate = certificate;
    }

    public String getCertificateFileName() {
        return certificateFileName;
    }

    public void setCertificateFileName(String certificateFileName) {
        this.certificateFileName = certificateFileName;
    }

    public String getCertificatePassword() {
        return certificatePassword;
    }

    public void setCertificatePassword(String certificatePassword) {
        this.certificatePassword = certificatePassword;
    }

    public String getJwtKey() {
        return jwtKey;
    }

    public void setJwtKey(String jwtKey) {
        this.jwtKey = jwtKey;
    }
}

File: ASP.NET Core version/Models/ConstantsData.cs
using ArangoDB.Client;

namespace Thalassa.Models
{
    // This class handles essential data (stored in database) needed during service initialization.
    [CollectionProperty(CollectionName = "Constants")]
    public class ConstantsData
    {
        [DocumentProperty(PropertyName = "certificate")]
        public string Certificate { get; set; }
        
        [DocumentProperty(PropertyName = "certificateFileName")]
        public string CertificateFileName { get; set; }
        
        [DocumentProperty(PropertyName = "certificatePassword")]
        public string CertificatePassword { get; set; }
        
        [DocumentProperty(PropertyName = "jwtKey")]
        public string JwtKey { get; set; }
    }
}